src/main/kotlin/io/snyk/plugin/ui/toolwindow/panels/SuggestionDescriptionPanelFromLS.kt

```diff
@@ -48,20 +49,25 @@ class SuggestionDescriptionPanelFromLS(
 
     init {
         if (pluginSettings().isGlobalIgnoresFeatureEnabled && issue.additionalData.details != null) {
-            val (jbCefBrowser, jbCefBrowserUrl)  = getJBCefBrowserIfSupported()
-            if (jbCefBrowser == null) {
+            val openFileLoadHandlerGenerator = OpenFileLoadHandlerGenerator(snykCodeFile)
+            val jbCefBrowserComponent  = getJBCefBrowserComponentIfSupported(issue.additionalData.details, { openFileLoadHandlerGenerator.generate(it) })
+            if (jbCefBrowserComponent == null) {
                 val statePanel = StatePanel(SnykToolWindowPanel.SELECT_ISSUE_TEXT)
                 this.add(wrapWithScrollPane(statePanel), BorderLayout.CENTER)
                 SnykBalloonNotificationHelper.showError(unexpectedErrorMessage, null)
             } else {
-                val panel = JPanel()
-                panel.add(jbCefBrowser.component, BorderLayout())
+                val lastRowToAddSpacer = 5
+                val panel = JPanel(
+                    GridLayoutManager(lastRowToAddSpacer + 1, 1, Insets(0, 10, 20, 10), -1, 20)
+                ).apply {
+                    this.add(jbCefBrowserComponent, panelGridConstraints(1))
+                }
                 this.add(
-                    wrapWithScrollPane(panel),
+                    panel,
                     BorderLayout.CENTER
                 )
-
-                jbCefBrowser.loadHTML(issue.additionalData.details, jbCefBrowserUrl)
+                this.add(panel)
+                // TODO: bottom panel
```

Comment:
@bastiandoetsch Do you happen to know what the bottom panel in https://github.com/snyk/snyk-intellij-plugin/blob/e20b59ccffac957d4b523351574c2b978d847362/src/main/kotlin/io/snyk/plugin/ui/toolwindow/panels/IssueDescriptionPanelBase.kt#L38-L43 is? I realised I forgot to add it to the panel but looking at the code it doesn't look like it's adding anything

Reply:
I don't think there's much happening below the example commit fixes. So it may be just a spacer or unnecessary.